src/lsdistcc.c

```diff
@@ -743,10 +745,11 @@ static int one_poll_loop(struct rslave_s* rs, struct state_s states[],
             nwithtries[sp->ntries]++;
             if (opt_verbose > 0)
                 fprintf(stderr,
-                        "now %ld %ld: Resending %s because "
-                        "deadline was %ld %ld\n",
-                        now.tv_sec, (long) now.tv_usec/1000, sp->req.hname,
-                        sp->deadline.tv_sec, (long) sp->deadline.tv_usec/1000);
+                        "now %lld %ld: Resending %s because "
+                        "deadline was %lld %ld\n",
+                        (long long) now.tv_sec, (long) now.tv_usec/1000,
+                        sp->req.hname, sp->deadline.tv_sec,
```

Comment:
missed a cast to (long long) here

Reply:
Actually not, since `tv_usec` is defined as a `suseconds_t` which is a `long` so it's correct as you've merged it. Thanks for that!